metrics.yaml

```diff
@@ -0,0 +1,15 @@
+# Required to indicate this is a `metrics.yaml` file
+$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+timeline:
```

Comment:
This file will eventually be defined in collaboration with @kirill-demtchouk 's team